cookbook-react/package.json

```diff
@@ -35,7 +35,8 @@
     "lint-fix": "./node_modules/eslint/bin/eslint.js src --fix && ./node_modules/stylelint/bin/stylelint.js '**/*.scss' --fix",
     "lint-scss": "./node_modules/stylelint/bin/stylelint.js '**/*.scss'",
     "lint-diff": "git diff --name-only --cached --relative --diff-filter=ACM | grep \\.js$ | xargs eslint",
-    "coverage-diff": "rescripts test --env=jsdom --coverage --watchAll=false --changedSince=development"
+    "coverage-diff": "rescripts test --env=jsdom --coverage --watchAll=false --changedSince=development",
+    "sass": "node ./scripts/compile-sass.js"
```

Comment:
Shouldn't this be in the `cookbook-web` folder/proyect?

Reply:
Yeah, but the cookbook-web doens't have a package.json. I decided to put it in the `app/` folder so it was in the top-level project, but for some reason when the husky hooks run they run from the cookbook-react project and not from the general one, regardless of which directory I'm in. Idk why that happens, we should check that out sometime soon. I just wanted to fix this quickly because we weren't compiling the scss